Gemfile

```diff
@@ -38,8 +38,6 @@ gem 'omniauth-facebook', '~> 9.0'
 gem 'omniauth-google-oauth2', '~> 1.1', '>= 1.1.1'
 gem 'omniauth-rails_csrf_protection'
 
-gem 'kaminari'
```

Comment:
Gem also need to be removed from gemfile.lock.
Run bundle install locally

Reply:
Already done.